### ISSUE TITLE
When having Parent and inherited Context on migration execution getting error: Unable to find table ...

### ISSUE THREAD

**Issue by borisdj** [isukces/EfCore.Shaman#17]
If part of application is in another project (class library), shared part that is used by multiple application.
Then in that shared project is BaseContext with shared Entities and DbSets, and in Main project is AppContext: BaseContext with others.
But when running migration at end of Up method on FixMigrationUp getting error:

> Unable to find table Company

It seems that when going through all DbSets the ones from Base class are not included in list and they should be for situations like this.

**Comment by isukces** [comment]
Thank you for that information. I will check this.

**Comment by borisdj** [comment]
I have managed to sort this out.
It wasn't a bug but the way where I have placed method **GetShamanOptions().**
I have added `public static ShamanOptions GetShamanOptions()` method in `BaseContext `but not in `AppContext `and that caused the problem.
I thought that configuring in parent would be enough but obviously not.
Now that I have moved the method to `AppContext` it works.
So I will close the issue.
